apps/src/aichat/views/chatWorkspace.module.scss

```diff
@@ -114,6 +114,6 @@ $tabs-default-spacing: 4px;
 }
 
 .modeDropdown {
-  margin: 0 0 10px 10px;
+  margin-left: 10px;
```

Comment:
it is safe to assume for now that the mode dropdown and suggested prompts won't appear at the same time. This position of the dropdown is not the final position, so it didn't seem worth doing extra logic to include a bottom margin if there are suggested prompts.